src/field_theory/adjoin.lean

```diff
@@ -92,6 +92,17 @@ noncomputable def bot_equiv : (⊥ : intermediate_field F E) ≃ₐ[F] F :=
   bot_equiv (algebra_map F (⊥ : intermediate_field F E) x) = x :=
 alg_equiv.commutes bot_equiv x
 
+noncomputable instance algebra_over_bot : algebra (⊥ : intermediate_field F E) F :=
```

Comment:
I was thinking that you could do this for an arbitrary intermediate field, just as with subalgebras

Reply:
It's going the wrong way around for that, I think

Reply:
It's not true that F is an algebra over any `K : intermediate_field F E`

Reply:
Ooh, sorry, I had the wrong idea for what this was doing... in my mind `F` was an extension of `E`...
Ok, then it should of course be like this.